aii-ks/src/main/perl/ks.pm

```diff
@@ -566,6 +566,7 @@ sub ksuserscript
 
     return unless $config->elementExists ($path);
     my $url = $config->getElement ($path)->getValue;
+    $url =~ s{LOCALHOST}{LOCALHOST}e;
```

Comment:
my recommnedation to use this form was more sarcasm wrt perl then genuine advise. i'm glad @jrha doesn't find this confusing 😄

Reply:
No more confusing than anything else in Perl...